src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerTest.java

```diff
@@ -283,4 +291,60 @@ void GIVEN_system_config_with_small_periodic_interval_WHEN_interval_elapses_THEN
         }
         assertEquals(1, allFiles.size());
     }
-}
+
+    @Test
+    void GIVEN_component_configs_WHEN_individual_configs_are_reset_THEN_correct_default_values_are_used()
+            throws Exception {
+        Path LOG_FILE_DIRECTORY_PATH_DEFAULT = tempRootDir.resolve("logs");
```

Comment:
nit: if this is only used once, it doesn't need to be an all-caps constant. You can also define it closer to where it's used

Reply:
changed and moved.